SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-cloudtest.md

```diff
@@ -80,4 +80,4 @@ When you’re done shooting your friends, you can click the **Stop** button in t
 
 
 <br/>------<br/>
-_2019-03-20 Page updated with limited editorial review_
\ No newline at end of file
+_2019-08-02 Page updated with limited editorial review_
```

Comment:
```suggestion
_2019-08-02 Page updated with limited editorial review: updated project name._</br>
_2019-03-20 Page updated with limited editorial review._
```

Reply:
file is out of date so will commit your suggestions manually.